plugins/org.wso2.developerstudio.eclipse.gmf.esb.persistence/src/org/wso2/developerstudio/eclipse/gmf/esb/internal/persistence/StoreMediatorTransformer.java

```
@@ -17,23 +17,30 @@
 package org.wso2.developerstudio.eclipse.gmf.esb.internal.persistence;
 
 import java.util.List;
+import java.util.Map;
 
 import org.apache.commons.lang.StringUtils;
 import org.apache.synapse.endpoints.Endpoint;
 import org.apache.synapse.mediators.base.SequenceMediator;
+import org.apache.synapse.util.xpath.SynapseXPath;
 import org.eclipse.core.runtime.Assert;
 import org.eclipse.emf.ecore.EObject;
+import org.jaxen.JaxenException;
 import org.wso2.developerstudio.eclipse.gmf.esb.EsbNode;
+import org.wso2.developerstudio.eclipse.gmf.esb.NamespacedProperty;
 import org.wso2.developerstudio.eclipse.gmf.esb.StoreMediator;
+import org.wso2.developerstudio.eclipse.gmf.esb.StoreMediatorSpecifyType;
 import org.wso2.developerstudio.eclipse.gmf.esb.persistence.TransformationInfo;
 import org.wso2.developerstudio.eclipse.gmf.esb.persistence.TransformerException;
 
 public class StoreMediatorTransformer extends AbstractEsbNodeTransformer {
 
-	public void transform(TransformationInfo information, EsbNode subject)
-			throws TransformerException {
-		information.getParentSequence().addChild(createStoreMediator(subject));
-		// Transform the property mediator output data flow path.
+	public void transform(TransformationInfo information, EsbNode subject) throws TransformerException {
+		try {
+			information.getParentSequence().addChild(createStoreMediator(subject));
+		} catch (JaxenException e) {
+			new TransformerException(e);
```

I guess we need to throw the exception here.